Makefile

```diff
@@ -172,6 +172,21 @@ install-python-linter:
 python-lint:
 	scripts/lint.sh
 
+## install-python-formatter: Install python formatter
+install-python-formatter:
+	pip install black==20.8b1
```

Comment:
1. Why hardcoded version here is bad?
2. Especially because it's beta and the package has no guarantees about backward compatibility we should have a hardcoded version here.

Reply:
I thought it would be better to, at least, upgrade to a stable version. And if there were mistakes in it, it would be great to have them fixed.

Reply:
We will upgrade it manually because of (2).
In general, a lot of software that claims that it's beta is much more stable than other ones that claim that their software is "stable".
So, "beta" here means that there's not backward compatibility garantuees. Because of this it's better to upgrade it manually with having hardcoded version.

Reply:
Oh, OK then.